Makefile

```diff
@@ -1,18 +1,34 @@
-BUILD_NUMBER := 1
-CP_VERSION := 3.3.0-SNAPSHOT
+# You can override vars like REPOSITORY in a local.make file
+-include local.make
```

Comment:
And is this something we really want or should we just have them overridden via envvars?